lnrpc/README.md

```diff
@@ -97,3 +97,32 @@ description):
 ```bash
 $ go get -u github.com/lightningnetwork/lnd/lnrpc
 ```
+
+## Generate protobuf definitions
+
+1. Download [v.3.4.0](https://github.com/google/protobuf/releases/tag/v3.4.0) of
```

Comment:
Just wondering, is there any reason you choose v3.4.0 instead of the latest v3.5.1 release?

Reply:
That's what's recommended in the [code contribution guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#Protobuf).